src/components/Dropdown/Dropdown.md

```diff
@@ -113,21 +90,16 @@ const text = 'The quick brown fox jumped over the lazy dog.';
 const options = [
   { key: 'section1', text: 'Icon, Label and Text', itemType: DropdownMenuItemType.Header},
   { key: 'A', icon: GroupIcon, label: 'Group', text: 'Option A' },
-  { key: 'B', icon: PhoneIcon, label: 'Phone', text: 'Option B' },
-  { key: 'C', icon: AttachIcon, label: 'Attachment', text: `${text} ${text} ${text} ${text}` },
+  { key: 'B', icon: AttachIcon, label: 'Attachment', text: `${text} ${text} ${text} ${text}` },
   { key: 'divider1', text: 'Section', itemType: DropdownMenuItemType.Divider},
   { key: 'section2', text: 'Label and Text', itemType: DropdownMenuItemType.Header},
-  { key: 'D', label: 'Group', text: 'Option D' },
-  { key: 'E', label: 'Group', text: 'Option E' },
-  { key: 'F', label: 'Group', text: `${text} ${text} ${text} ${text}` },
```

Comment:
Removed some items, so that the callout could open within the vertical space of the viewport.